config/env/default.js

```diff
@@ -43,8 +43,8 @@ module.exports = {
   logo: 'modules/core/client/img/brand/logo.png',
   favicon: 'modules/core/client/img/brand/favicon.ico',
   uploads: {
-    profileUpload: {
-      dest: './modules/users/client/img/profile/uploads/', // Profile upload destination path
+    profile: {
```

Comment:
Just a sidenote: should config changes be documented at changelog within the PR so that you ensure mentioning them when a new release goes out? (ping @lirantal)